lib/utils/testLauncherHelper.js

```diff
@@ -315,16 +315,16 @@ async function runTestOnDevice(cmdArgv, ownArgv, device, runMode, ipcPort) {
 /**
  * Prepares environment for the child process.
  *
- * @param {string} deviceId - id of the device
+ * @param {string} device - device object. deviceId and config should be provided.
  * @param {number} port - ipc port
  * @returns {{shell: true, env: Object}}
  */
-function getChildOptions(deviceId, port) {
+function getChildOptions(device, port) {
 	return {
 		shell: true,
 		env: {
 			...process.env,
-			[envVars.SUITEST_CHILD_PROCESS]: `${deviceId}|${port}`,
+			[envVars.SUITEST_CHILD_PROCESS]: `${device.deviceId}|${device.config}|${port}`,
```

Comment:
pls ensure that this change will not brake our exemples of using different reporters with suitest, need to check that all commands works as was before that uses `SUITEST_CHILD_PROCESS` from https://github.com/SuitestAutomation/suitest-js-api-mocha-demo/blob/master/package.json.